### PR TITLE
Core: add notification for client unable to join full room

### PR DESCRIPTION
**Pull request by kevinwhereby**
-------

### Description
Lets the host know if another user tries to join a full room
**Summary:**

**Related Issue:**
https://linear.app/whereby/issue/COB-972/add-an-sdk-host-notification-when-someone-tries-to-join-a-full-room

### Testing
1. yarn build
1. yarn dev:sample-app
1. go to the sample app page and join a P2P room as a host
1. join the same room with 3 more clients in other browsers
1. try to join from a fifth browser
1. see the notification in the sample app 

### Screenshots/GIFs (if applicable)
image

### Checklist

-   [x] My code follows the project's coding standards.
-   [x] Prefixed the PR title and commit messages with the service or package name
-   [x] I have written unit tests (if applicable).
-   [x] I have updated the documentation (if applicable).
-   [x] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

**Comment by nandito** [comment]
I followed the test steps but I haven't seen the notification. I ran `rm -rf node_modules`, then `yarn` `yarn build` `yarn dev:sample-app`.  I see the `client_unable_to_join` signal evt in devtools when the fifth knocker's req is accepted. There's no (related) console error or warning.

What did I do wrong? 😕

File: packages/media/src/utils/types.ts [hunk 1]
```diff
@@ -184,6 +184,13 @@ export interface ClientMetadataReceivedEvent {
     };
 }
 
+export interface ClientUnableToJoinEvent {
```
Comment (nandito) [inline packages/media/src/utils/types.ts:187]: the `client_unable_to_join` is a signal event afaik, why do we need it in the `media` utils? 🤔

Comment (kevinwhereby) [reply packages/media/src/utils/types.ts:187]: That's where the rest of the event payload types are I think

Comment (kevinwhereby) [reply packages/media/src/utils/types.ts:187]: ah but I did forget to add it to the bigger list of signal events 💡

**Pull request merged**